float64 fallen_status
float64[] orientation
float64[] pos
float64 distance_minimum
float64[] rpy_vel
float64[] vel
float64 sim_time
